crates/ruff_linter/src/rules/flake8_bandit/rules/hardcoded_sql_expression.rs

```diff
@@ -88,8 +95,6 @@ pub(crate) fn hardcoded_sql_expression(checker: &mut Checker, expr: &Expr) {
             };
             string.value.to_str().escape_default().to_string()
         }
-        // f"select * from table where val = {val}"
-        Expr::FString(f_string) => concatenated_f_string(f_string, checker.locator()),
```

Comment:
I don't think we should avoid checking f-strings otherwise we're reducing the surface area for the rule. The fix should be to just remove the `escape_default` call but we still need to concatenated the f-strings.

The reason that string literals aren't being concatenated explicitly is because the `to_str` call above (`string.value.to_str()`) will concatenated them internally. 

Does this make sense?